flow/workflows/cdc_flow.go

```diff
@@ -568,3 +574,15 @@ func CDCFlowWorkflow(
 		}
 	}
 }
+
+func GetFlowMetadataContext(ctx workflow.Context, flowJobName string, sourceName string, destinationName string) (workflow.Context, error) {
+	metadataCtx := workflow.WithLocalActivityOptions(ctx, workflow.LocalActivityOptions{
+		StartToCloseTimeout: 10 * time.Second,
```

Comment:
give more time here, local activity has no heartbeat

Reply:
I think I can put in 30 secs